src/js/build.sh

```diff
@@ -12,8 +12,7 @@ set -ex
 root=$(git rev-parse --show-toplevel)
 js_root="${root}/src/js/"
 cd "$js_root"
-build_dir="${V_BUILD_DIR:-build}/node/output"
-build_dir_base="${root}/${build_dir}"
+build_dir_base="${root}/build/node/output"
```

Comment:
@andresaristizabal please review

Reply:
Hi, this is fine, with the new task approach, this script should save the artifacts on `redpanda_folder/build/node/output`, but you remove the `V_BUILD_DIR` variable, that variable allows to user change the path for building, the new task approach doesn't allow change the build path?

Reply:
is this script intended to be executed by devs? can it be replaced by https://github.com/vectorizedio/vtools/pull/147 ?